POEApi.Model/Gem.cs

```diff
@@ -6,13 +6,41 @@ namespace POEApi.Model
     public class Gem : SocketableItem
     {
         public int Level { get; set; }
+        public double LevelProgress { get; private set; }
+        public int Numerator { get; set; }
```

Comment:
Update the names here to reflect that they are the numerator and denominator that relate to the gem's current experience.  Adding comments with more detail wouldn't hurt, either.